### PR TITLE
[TASK] Switch to using ripple-lib-transactionparser to compute transaction balance changes

### PR DESCRIPTION
**Pull request by clark800**
Also a change in testutils to make debugging JSON diffs easier.

### CODE REVIEW

**Comment by clark800** [comment]
This PR is ready for review

File: package.json [hunk 1]
```diff
@@ -36,11 +36,13 @@
     "nconf": "~0.6.9",
     "node-uuid": "~1.4.1",
     "ripple-lib": "^0.10.1-rc2",
+    "ripple-lib-transactionparser": "^0.1.0",
     "sqlite3": "~3.0.2",
     "supertest": "^0.13.0",
     "winston": "^0.7.3"
   },
   "devDependencies": {
+    "assert-diff": "*",
```
Comment (geertweening) [inline package.json]: That's cool

**Comment by geertweening** [comment]
please rebase, otherwise LGTM

**Comment by clark800** [comment]
Ok rebased

**Pull request merged**